src/build.c

```diff
@@ -1057,13 +1057,13 @@ gboolean build_parse_make_dir(const gchar *string, gchar **prefix)
 	if (string == NULL)
 		return FALSE;
 
-	if ((pos = strstr(string, "Entering directory")) != NULL)
+	if ((pos = strstr(string, "Entering directory `")) != NULL)
```

Comment:
Changing the string in the if-conditional could break something later.

Reply:
Yeah, so just rescan from the start of "Enetring Directory" and it doesn't matter if the string changes.

Reply:
Done!